conf/airframes/AGGIEAIR/aggieair_rp3_lia.xml

```diff
@@ -35,6 +35,8 @@ RP3 Lisa MX
 
     <define name="AGR_CLIMB" />
     <define name="POLY_OSAM_HALF_SWEEP_ENABLED" value="FALSE"/>
+    <define name="AIRSPACE"/>
+    <define name="DATALINK_LOST_TIME" value="45"/>
```

Comment:
Since there is already a check of datalink loss I think this naming should be more clear that it is for the home mode.